GramAddict/plugins/interact_hashtag_posts.py

```diff
@@ -0,0 +1,264 @@
+import logging
+from functools import partial
+from random import seed, shuffle
+
+from colorama import Fore, Style
+from GramAddict.core.decorators import run_safely
+from GramAddict.core.device_facade import DeviceFacade
+from GramAddict.core.filter import Filter
+from GramAddict.core.interaction import (
+    _on_interaction,
+    _on_like,
+    _on_watch,
+    interact_with_user,
+    is_follow_limit_reached_for_source,
+)
+from GramAddict.core.plugin_loader import Plugin
+from GramAddict.core.scroll_end_detector import ScrollEndDetector
+from GramAddict.core.storage import FollowingStatus
+from GramAddict.core.utils import get_value, random_sleep
+from GramAddict.core.views import (
+    TabBarView,
+    HashTagView,
+    OpenedPostView,
+    PostsViewList,
+    SwipeTo,
+)
+
+logger = logging.getLogger(__name__)
+
+# Script Initialization
+seed()
+
+
+class InteractHashtagLikers(Plugin):
+    """Handles the functionality of interacting with a hashtags post owners"""
+
+    def __init__(self):
+        super().__init__()
+        self.description = (
+            "Handles the functionality of interacting with a hashtags post owners"
+        )
+        self.arguments = [
+            {
+                "arg": "--hashtag-posts-recent",
```

Comment:
can we also add support for hashtag-posts-top? Should be same as this, but choose the other tab. Don't know why someone would want it, but 🤷

Reply:
done